e2e/src/landing-page/landing-page.e2e-spec.ts

```diff
@@ -1,16 +1,18 @@
-import { AppPage } from './app.po';
+import { LandingPage } from './landing-page.po';
 import { browser, logging } from 'protractor';
 
-describe('workspace-project App', () => {
-  let page: AppPage;
+describe('LandingPage', () => {
+  let page: LandingPage;
 
   beforeEach(() => {
-    page = new AppPage();
+    page = new LandingPage();
   });
 
-  it('should display welcome message', () => {
-    page.navigateTo();
-    expect(page.getTitleText()).toEqual('admin-web-ui app is running!');
+
+
+  it('should display landing page if user navigates to /login', () => {
```

Comment:
Good test. I've raised an issue to rename the "landing-page" as a login page #4 .